exotations/dynamics_solvers/exotica_cartpole_dynamics_solver/test/test_cartpole_diff.py

```diff
@@ -0,0 +1,62 @@
+import autograd.numpy as np
```

Comment:
What is the pip command for this? I couldn't find a rosdep key for it that we could add to the package.xml.

Reply:
It needs to be a valid key that you can query with `rosdep resolve ...`. It currently isn't upstream. I will have a look at it.

Reply:
I've requested inclusion upstream with the key `autograd-pip` - cf. https://github.com/ros/rosdistro/pull/21332.

Reply:
Changed it to finite differences now and added comments when this gets merged.